src/test.rs

```diff
@@ -158,6 +158,53 @@ fn cross_process_embedded_senders() {
     assert_eq!(received_person, person);
 }
 
+#[cfg(not(any(feature = "force-inprocess", target_os = "windows", target_os = "android", target_os = "ios")))]
+#[test]
+fn stress_in_process() {
+    enum Message {
+        CloneApi(IpcSender<i32>),
+    }
+
+    impl Serialize for Message {
```

Comment:
I'm not very familiar with Serde, so this might be a silly question: but why do you need to implement these manually, rather than just deriving?